src/ResourceManager/MarketplaceOrdering/ChangeLog.md

```diff
@@ -0,0 +1,26 @@
+﻿<!--
+    Please leave this section at the top of the change log.
+
+    Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
+
+    ## Current Release
+    * Overview of change #1
+        - Additional information about change #1
+    * Overview of change #2
+        - Additional information about change #2
+        - Additional information about change #2
+    * Overview of change #3
+    * Overview of change #4
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Current Release
+
+## Version 1.0.0
```

Comment:
@prbansa please remove this header and put all information under the `Current Release` header. We have a script that runs during the release that will place all of this information under the correct header.

Reply:
Done